### ISSUE TITLE
Required Billing Fields

### ISSUE THREAD

**Issue by embedded1** [JDBurnZ/pyminfraud#3]
I'm using your library for validating orders processed via PayPal and therefore I don't have any billing information.

I've talked to MaxMind and they told me I can continue without the billing data.
Why did you mark the billing as required fields?

Can you please mark the billing fields as not required and push an update?

**Comment by JDBurnZ** [comment]
Good catch, I was with the understanding this was a required field. I'll review and update as appropriate.

**Comment by embedded1** [comment]
thanks for your fast reply.
I will update and verify that your fix works.

**Comment by JDBurnZ** [comment]
Fixed, change was made in pyminfraud/fields.py.

**Comment by embedded1** [comment]
did you update git?
why don't you support pip?

**Comment by JDBurnZ** [comment]
Yes, updated in Git.
I thought I had support for pip working at one time, but I see it's broken now.. I'll re-visit when I get a chance.

**Comment by embedded1** [comment]
thanks, just updated, will keep you posted
